### PR TITLE
Added functionality

### PR DESCRIPTION
**Pull request by ekenecf**
Added basic functionality using JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,12 +32,15 @@
...
+                        <li class="link"> <a href="#portfolio">Portfolio</a></li>
+                        <li class="link"> <a href="#About">About</a></li>
+                        <li class="link"> <a href="#contact">Contact</a> </li>
                     </ul>
-                    <button class="menu"><i class="fa-solid fa-bars"></i></button>
                 </nav>
+                <div class="hamburger">
+                    <button class="close"><i class="fa-solid fa-xmark"></i></button>
+                </div>
+                <button class="menu"><i class="fa-solid fa-bars"></i></button>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix the navbar view on the desktop and mobile view also. You can see, that on the desktop version, there are no nav links:
Screen Shot 2022-03-08 at 19 11 08

- [ ] Your mobile menu is not cover the full page. You can also see the logo on top of the mobile menu. Considering the Figma template the mobile menu should not consist of the logo and should cover the full page:


> Here is the more correct version of the menu displaying:



> Here is also the Figma template:
Screen Shot 2022-03-08 at 19 17 25

- [ ] Additionally, please, consider fixing the `@media only screen and (min-width: 992px) ` - 992px should be the board for switching from desktop to mobile version and the hamburger icon should appear. In your version it does not happen on 992px, but happened on 1050px:

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @ekenecf.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider setting the JS linter also. Here you can find the [linters.yml](https://github.com/elmar8287/leaderboard/blob/dev/.github/workflows/linters.yml) file, how it should be in the correct way.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ekenecf) [reply index.html]: Hello @elmar8287, as requested by you. I have:
1.  set the JS linter,
2.  Fixed the navbar view on the desktop and mobile view also,
3.  Made the mobile menu to cover the full page,
4.  Fixed the @media only screen and (min-width: 992px). Kindly check to confirm.
   
    Thank you

File: index.html [hunk 2]
```diff
@@ -31,13 +31,17 @@
...
+                    <div class="hamburger">
+                        <button class="close"><i class="fa-solid fa-xmark"></i></button>
+                    </div>
                     <ul class="nav-list">
-                        <li> <a href="#">Portfolio</a> </li>
-                        <li> <a href="#">About</a> </li>
-                        <li> <a href="#">Contact</a> </li>
+                        <li class="link"> <a href="#portfolio">Portfolio</a></li>
+                        <li class="link"> <a href="#About">About</a></li>
+                        <li class="link"> <a href="#contact">Contact</a> </li>
```
Comment (BenMukebo) [inline index.html]: - [ ] Great work on implementing changes as requested by the previous reviewer, but I can see that you make some on them in half, considering the `gift screen record` he left, the navigation menu should take the full page, neither the user scrolls it, but it is not what happening now, see this image below, onScroll, the nav menu moves together with the page, which is not correct, please fix this and your project will be good to go.
- Giving the `position: fixed;` to the nav menu may solve the problem
- [ ] 
nav

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@ekenecf,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


### To highlight 💪🏻

- No linter errors. ✔️ 
- Githubflow was correctly used.  ✔️ 
- Good commit messages ✔️
 👍  Good job!

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ekenecf) [reply index.html]: Hi @BenMukebo,
As requested by you, I have:
1. Made the navigation menu to take the full page and,
2.  Made the nav menu stop moving together with the page.

**Review by ReshyResh — APPROVED** [review]
# Hi @ekenecf , 

### Good job addressing the changes requested by the previous reviewer.
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 

#### [OPTIONAL]

- [x] Please consider calling the function to toggle the menu once the user clicks on any of the options, this will improve the UX.


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**